### DISCUSSION TITLE
Feedback on `/docs/variables/`

### DISCUSSION THREAD

**Discussion by Juliakas** [iTwin/iTwinUI#2304] [Feedback]
Allow easilly switching dark, light theme on https://itwinui.bentley.com/docs page. This would make reviewing variables extra helpful

**Comment by mayank99** [comment]
Thanks for the feedback. This is something we're planning to work on. In the meantime, you can look at https://itwin.github.io/iTwinUI/css/color

**Reply by Juliakas** [reply]
Thanks for this reference, will bookmark it.
